partition/bootloader.go

```diff
@@ -146,3 +155,13 @@ func MarkBootSuccessful(bootloader Bootloader) error {
 
 	return bootloader.SetBootVars(m)
 }
+
+func reboot(afterMins int) error {
+	cmd := exec.Command("shutdown", fmt.Sprintf("+%d", afterMins), "-r", shutdownMsg)
+	if out, err := cmd.CombinedOutput(); err != nil {
+		logger.Noticef("%s", osutil.OutputErr(out, err))
```

Comment:
this should return osutil.OutputErr directly and the logging moved back to daemon.go I think